src/ui/QuoteMessage/utils.ts

```diff
@@ -0,0 +1,73 @@
+import {CoreMessageType, isFileMessage, isMultipleFilesMessage} from "../../utils";
+import {match} from "ts-pattern";
+import {FileMessage, MultipleFilesMessage, UploadedFileInfo} from "@sendbird/chat/message";
+import {Thumbnail} from "@sendbird/chat/lib/__definition";
+
+export function getMessageFirstFileType(message: CoreMessageType): string {
+  return match(message)
+    .when(m => isFileMessage(m), () => {
+      return (message as FileMessage)?.type ?? '';
+    })
+    .when(m => isMultipleFilesMessage(m), () => {
+      const castedMessage: MultipleFilesMessage = message as MultipleFilesMessage;
+      return getFirstFileInfo(castedMessage)?.mimeType ?? '';
+    })
+    .otherwise(() => {
+      return '';
+    });
+}
+
+function getFirstFileInfo(message: MultipleFilesMessage): UploadedFileInfo {
```

Comment:
```suggestion
function getFirstFileInfo(message: MultipleFilesMessage): UploadedFileInfo | null {
```
It seems it returns `null` too.

Reply:
thanks for the catch!